src/_utils/index.tsx

```diff
@@ -1,12 +1,18 @@
 import React, { Fragment } from 'react'
 import { canUseDOM } from 'exenv'
 
-export const prefix = (
-  modifiers: Classcat.ClassObject = {},
-  baseClass: string = 'kirk',
-): string => {
+const DEFAULT_CLASS_PREFIX = 'kirk'
+
+export const prefix = (modifiers: Classcat.ClassObject, baseClass?: string): string => {
```

Comment:
The `modifiers`  parameter is no longer optional (it was never called as `prefix()` anyway)